ui/app/AppLayouts/Profile/views/AppearanceView.qml

```diff
@@ -13,25 +12,21 @@ import StatusQ.Core 0.1
 import StatusQ.Core.Theme 0.1
 import StatusQ.Controls 0.1 as StatusQ
 
-import "../popups"
-import "../stores"
-
 SettingsContentBase {
     id: appearanceView
 
+    required property var localSettings
```

Comment:
I guess a further improvement would be to pass the individual settings instead. It would be more explicit and have type protection

Reply:
I agree with @jrainville, ideally the `theme` and `fontSize` should be taken by individual, required, typed properties. And the component should report intention of change by `themeChangeRequested` and `fontSizeRequested` signals. Than this component is pure UI component, with no internal responsibility of setting those things in a specific way. Than `updateTheme` and `updateFontSize` methods are not need there.

Reply:
Alright, will change the theme/fontSize handling, and move the `Settings` to AppMain

Reply:
@micieslak @jrainville theme handling moved to AppMain; pls have a second look :beers: